tests/cxx/unit_tests/ghostfragment/testing/hydrocarbon.hpp

```diff
@@ -0,0 +1,153 @@
+    ethane.push_back(
+      chemist::Atom("H", 1, 1837.289, 1.249461579, 1.512120398, -0.8901392951));
+    ethane.push_back(
+      chemist::Atom("H", 1, 1837.289, 1.249461579, 1.512120398, 0.8901392951));
+
+    // Last Hydrogen
+    ethane.push_back(
+      chemist::Atom("H", 1, 1837.289, 2.139600874, 0.2539438837, 0));
+
+    return ethane;
+}
+
+// Propane Molecule (3 carbons)
+chemist::Molecule Propane() {
+    chemist::Molecule propane;
+
+    // Carbon Atoms
+    propane.push_back(chemist::Atom("C", 6, 21874.662, 0, 0, 0));
+    propane.push_back(
+      chemist::Atom("C", 6, 21874.662, 1.249461579, 0.8830321408, 0));
+    propane.push_back(chemist::Atom("C", 6, 21874.662, 2.498923159, 0, 0));
+
+    // First Hydrogen
+    propane.push_back(
+      chemist::Atom("H", 1, 1837.289, -0.8901392951, 0.6290882571, 0));
+
+    // Side Hydrogens
+    propane.push_back(
+      chemist::Atom("H", 1, 1837.289, 0, -0.6290882571, -0.8901392951));
+    propane.push_back(
+      chemist::Atom("H", 1, 1837.289, 0, -0.6290882571, 0.8901392951));
+    propane.push_back(
+      chemist::Atom("H", 1, 1837.289, 1.249461579, 1.512120398, -0.8901392951));
+    propane.push_back(
+      chemist::Atom("H", 1, 1837.289, 1.249461579, 1.512120398, 0.8901392951));
+    propane.push_back(chemist::Atom("H", 1, 1837.289, 2.498923159,
+                                    -0.6290882571, -0.8901392951));
+    propane.push_back(chemist::Atom("H", 1, 1837.289, 2.498923159,
+                                    -0.6290882571, 0.8901392951));
+
+    // Last Hydrogen
+    propane.push_back(
+      chemist::Atom("H", 1, 1837.289, 3.389062454, 0.6290882571, 0));
+
+    return propane;
+}
+
+bool AreMoleculesEqual(chemist::Molecule m1, chemist::Molecule m2) {
+    int size1 = m1.size();
+    int size2 = m2.size();
+
+    // First checks that the molecules have the same number of atoms
+    if(size1 != size2) {
+        std::cout << "Size 1: " << size1 << " does not equal Size 2: " << size2
+                  << "." << std::endl;
+        return false;
+    }
+
+    // Grabs the nuclei of each molecule to check each atom individually
+    chemist::Molecule::const_nuclei_reference nuclei1 = m1.nuclei();
+    chemist::Molecule::const_nuclei_reference nuclei2 = m2.nuclei();
+
+    for(int i = 0; i < size1; i++) {
+        // Each atom must be checked against each other for name and position
+        chemist::Nuclei::const_reference atom1 = nuclei1.at(i);
+        chemist::Nuclei::const_reference atom2 = nuclei2.at(i);
+
+        // Checking the name
+        if(atom1.name() != atom2.name()) {
+            std::cout << "Name 1: " << atom1.name()
+                      << " is not equal to Name 2: " << atom2.name() << "."
+                      << std::endl;
+            return false;
+        }
+
+        // Checking coordinates using a tolerance of 8 digits
+        // X coordinate
+        if((atom1.x() - atom2.x()) > 0.0000001) {
```

Comment:
```suggestion
       REQUIRE(atom1.x() == Approx(atom2.x()));
```
See [here](https://github.com/catchorg/Catch2/blob/v2.x/docs/assertions.md#floating-point-comparisons) for more info on how Catch2 supports floating point comparisons.